documentation/content/en/books/handbook/boot/_index.adoc

```diff
@@ -103,7 +103,12 @@ Work is under way to provide direct UEFI booting.
 
 The code within the MBR is typically referred to as a _boot manager_, especially when it interacts with the user.
 The boot manager usually has more code in the first track of the disk or within the file system.
-Examples of boot managers include the standard FreeBSD boot manager boot0, also called Boot Easy, and Grub, which is used by many Linux(R) distributions.
+Examples of boot managers include the standard FreeBSD boot manager boot0, also called Boot Easy, and GNU GRUB, which is used by many Linux(R) distributions.
+
+[NOTE]
+====
+There's :faq:[a frequently asked question] about GRUB. Beyond the answer there, users of GRUB should please refer to https://www.gnu.org/software/grub/grub-documentation.html[GNU-provided documentation]. 
```

Comment:
Markup here is wrong. 

From <https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#links> for _links_, I can't tell how to fix the link to <https://docs.freebsd.org/en/books/faq/#grub-loader>.

Reply:
See https://docs.freebsd.org/en/books/fdp-primer/asciidoctor-primer/#asciidoctor-links and specifically https://docs.freebsd.org/en/books/fdp-primer/asciidoctor-primer/#asciidoctor-links-internal .